src/frequenz/sdk/timeseries/battery_pool/_metric_calculator.py

```diff
@@ -323,11 +323,14 @@ def calculate(
             ):
                 continue
 
+            usable_capacity = capacity * (soc_upper_bound - soc_lower_bound)
```

Comment:
It's a bit confusing that this is named `usable_capacity` when it is in fact just multiplied with with a number `<100`..

Reply:
But what else can we call it?

If rated capacity is 100kWh,  and soc bounds are 10-90%,  then usable capacity is 80kWh.  Because if we start discharging when the battery is full (i.e 90%) and stop when it is empty (i.e 10%), we'll get 80kWh out of it.

Maybe a comment with this example helps?

Reply:
oh you mean because it is multiplied by numbers > 1.  Yes,  that's just to do fewer calculations,  and it gets cancelled out in a later stage.

Reply:
`capacity_calculation_step`

Reply:
true, I've added a `_x100` prefix to all the related variables, and added a comment explaining what's going on.

Reply:
Very nice!